plugins/org.eclipse.elk.alg.force/src/org/eclipse/elk/alg/force/stress/StressLayoutProvider.java

```diff
@@ -37,9 +36,17 @@
     public void layout(final ElkNode layoutGraph, final IElkProgressMonitor progressMonitor) {
         progressMonitor.begin("ELK Stress", 1);
 
+
         // calculate initial coordinates
         if (!layoutGraph.getProperty(StressOptions.INTERACTIVE)) {
```

Comment:
Not negating the condition and swapping the two branches would make this a little more readable. I always have to think to figure out when the `else` branch is executed.